### DISCUSSION TITLE
How can Headless unit tests be associated with and run on the application under test?

### DISCUSSION THREAD

**Discussion by djyGitHubAccount** [AvaloniaUI/Avalonia#17279] [Q&A]
How can Headless unit tests be associated with and run on the application under test? The application being tested is running in Headless mode. Can the variables in the ViewModel be accessed?

**Comment by maxkatz6** [comment]
If you want to set a specific Application type for the headless tests (or specific BuildAvaloniaApp), you need to use AvaloniaTestApplication attribute.
https://docs.avaloniaui.net/docs/concepts/headless/headless-xunit#initialize-xunit-tests

Without this attribute, default Application will be created, without any themes.

**Comment by djyGitHubAccount** [comment]
这是来自QQ邮箱的自动回复邮件。
 
您好，我已收到你的邮件，我会尽快给您回复。
